dockers/skylab/create-npz-output/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7.2
```

Comment:
I know we haven't implemented the new docker best practices to the skylab dockers yet, but could you check that this docker follows those best practices? At least for new ones we should get in the habit of following them. You can see it here for now: https://github.com/broadinstitute/warp/pull/214/files#diff-ad74a7216f554360e9937b4155bfbd4860931507790552476c5cadcd461d1a96

Reply:
*same for all dockerfiles in this PR

Reply:
we followed the guidelines